test/e2e/cluster/manifests/addon-manager.json

```diff
@@ -0,0 +1,37 @@
+{
+	"apiVersion": "v1",
+	"kind": "Pod",
+	"metadata": {
+		"name": "kube-addon-manager",
```

Comment:
not at the moment, but we could in the future test the kube-dns cluster yamls as well as part of the e2e suite